app/enterprise/2.1.x/developer-portal/structure-and-file-types.md

```diff
@@ -207,7 +211,17 @@ x-headmatter:
 ...
 ```
 
-Specs are a collection meaning their `layout` and `route` are determined by the portal configuration and not the file itself. Specs are rendered by default by the `system/spec-renderer.html` layout, under the route pattern `/documentation/:name` where name is the name of the particular spec file. So a spec with a path of `specs/myfirstspec.json` renders in the portal as `/documentation/myfirstspec`.  If you want to overwrite the hardcoded spec collection config, you can do so by including your own in `portal.conf.yaml`. Check out the Collections section of our `Working with Templates` guide to learn more.
+Specs are a collection meaning their `layout` and `route` are determined by the
+portal configuration and not the file itself. Specs are rendered by default with
+the `system/spec-renderer.html` layout, under the route pattern `/documentation/:name`,
+where `:name` is the name of the particular spec file. So a spec with a path of
+`specs/myfirstspec.json` renders in the portal as `/documentation/myfirstspec`.  
+If you want to overwrite the hardcoded spec collection config, you can do so by
+including your own in `portal.conf.yaml`. Check out the Collections section of
+our `Working with Templates` guide to learn more.
+
+You can also use the [Portal Files API](/enterprise/{{page.kong_version}}/developer-portal/files-api)
```

Comment:
Added link here to new topic. The other changes above in this file were just to wrap char lines.